src/components/IdentityLine.tsx

```diff
@@ -47,6 +69,13 @@ const IdentityBox = styled.div`
   padding: ${({ theme }) => theme.spaces.component.xs};
 `;
 
+const IdentityInfo = styled.div`
+  p {
+    font-size: 70%;
```

Comment:
You should use theme value, and more importantly never use percentages for font size. We use rem for font sizes to handle zooming in the browser (`cmd +`/ `cmd -`)

Reply:
OK fixed fdd8618